secretmanager/src/test/java/secretmanager/SnippetsIT.java

```diff
@@ -218,14 +252,36 @@ public void testDisableSecretVersion() throws IOException {
   }
 
   @Test
-  public void testEnableSecretVersion() throws IOException {
+  public void testDisableSecretVersion() throws IOException {
+    String name = TEST_SECRET_VERSION_TO_DISABLE_WITH_ETAG.getName();
+    String etag = TEST_SECRET_VERSION_TO_DISABLE_WITH_ETAG.getEtag();
+    DisableSecretVersionRequest request =
+        DisableSecretVersionRequest.newBuilder().setName(name).setEtag(etag).build();
+    new DisableSecretVersion().disableSecretVersion(request);
+
+    assertThat(stdOut.toString()).contains("Disabled secret version");
+  }
+
+  @Test
+  public void testEnableSecretVersionWithName() throws IOException {
     SecretVersionName name = SecretVersionName.parse(TEST_SECRET_VERSION_TO_ENABLE.getName());
     new EnableSecretVersion()
         .enableSecretVersion(name.getProject(), name.getSecret(), name.getSecretVersion());
 
     assertThat(stdOut.toString()).contains("Enabled secret version");
   }
 
+  @Test
+  public void testEnableSecretVersion() throws IOException {
+    String name = TEST_SECRET_VERSION_TO_ENABLE_WITH_ETAG.getName();
+    String etag = TEST_SECRET_VERSION_TO_ENABLE_WITH_ETAG.getEtag();
+    EnableSecretVersionRequest request =
+        EnableSecretVersionRequest.newBuilder().setName(name).setEtag(etag).build();
+    new EnableSecretVersion().enableSecretVersion(request);
```

Comment:
nit: These functions should probably just all be static, so that this initialization isn't necessary.

Reply:
Done.